salt/utils/reactor.py

```diff
@@ -346,7 +346,7 @@ def caller(self, fun, *args, **kwargs):
         Wrap Caller to enable executing :ref:`caller modules <all-salt.caller>`
         '''
         log.debug("in caller with fun {0} args {1} kwargs {2}".format(fun, args, kwargs))
-        args = kwargs.get('args', [])
```

Comment:
i would rather see this changed to 
```
if args is None:
    args = kwargs.get('args', [])
else:
    args.extend(kwargs.get('args', []))
```

Because I believe that there is an instance where the args can be passed through kwargs.